### ISSUE TITLE
Inconsistent tab outline on calendar event popover

### ISSUE THREAD

**Issue by miaulalala** [nextcloud/calendar#4387]
### Steps to reproduce

1. Open a small calendar modal
2. Tab through it
3. Look at the tab outline for the calendar icon in the date field.

![image](https://user-images.githubusercontent.com/7427347/179819146-687a296c-81ee-4d04-becc-0738b92b663b.png)

### Expected behavior

The tab outline is styled properly, and has the same colour as the other tab outlines.

### Actual behaviour

The tab outline has a black colour and is too big for the icon.

### Calendar app version

3.5.0-alpha1

### CalDAV-clients used

Web

### Browser

Version 103.0.5060.114 (Official Build) (64-bit)

### Client operating system

Ubuntu

### Server operating system

Ubuntu

### Web server

Apache

### Database engine version

MySQL

### PHP engine version

PHP 8.0

### Nextcloud version

NC25

### Updated from an older installed version or fresh install

Updated from an older version

**Comment by SebastianKrupinski** [comment]
Closing this as resolved
